### PR TITLE
pre-release `tm_a_regression` document update

### PR DESCRIPTION
**Pull request by kartikeyakirar**
part of https://github.com/insightsengineering/teal.modules.general/issues/624

### CODE REVIEW

File: R/tm_a_regression.R [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-#' @param alpha optional, (`numeric`) If scalar then the plot points will have a fixed opacity. If a
-#'   slider should be presented to adjust the plot point opacity dynamically then it can be a vector of
-#'   length three with `c(value, min, max)`.
-#' @param size optional, (`numeric`) If scalar then the plot point sizes will have a fixed size
-#'   If a slider should be presented to adjust the plot point sizes dynamically then it can be a
-#'   vector of length three with `c(value, min, max)`.
-#' @param default_outlier_label optional, (`character`) The default column selected to label outliers.
-#' @param default_plot_type optional, (`numeric`) Defaults to Response vs Regressor.
+#' Response variables from an incoming dataset with filtering and selecting.
+#' @param alpha (`numeric`, optional) Specifies point opacity.
```
Comment (vedhav) [inline R/tm_a_regression.R:17]:
```suggestion
#' @param alpha (`integer(1)` or `integer(3)`, optional) Specifies point opacity.
```

File: R/tm_a_regression.R [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
+#' - If vector of `length == 1` then the plot points will have a fixed opacity.
+#' - while vector of `value`, `min`, and `max` allows dynamic adjustment.
+#' @param size (`numeric`, optional) Specifies point size.
```
Comment (vedhav) [inline R/tm_a_regression.R:20]:
```suggestion
#' @param size (`integer(1)` or `integer(3)`, optional) Specifies point size.
```

Comment (vedhav) [inline R/tm_a_regression.R:20]: Let us follow the same convention for the arg document what we use in the PR #669

Comment (vedhav) [reply R/tm_a_regression.R]: Can we have a similar wording as we did in that PR?

```r
#' - When the length of `bins` is one: The histogram bins will have a fixed size based on the `bins` provided.
#' - When the length of `bins` is three: The histogram bins are dynamically adjusted based on vector of `value`, `min`, and `max`.
```

Comment (vedhav) [reply R/tm_a_regression.R]: @kartikeyakirar After this change we can merge this one too.

Comment (kartikeyakirar) [reply R/tm_a_regression.R]: [9186292](https://github.com/insightsengineering/teal.modules.general/pull/655/commits/91862926e01ae4dae86c215fb588ce0354705578)

**Review by vedhav — APPROVED** [review]
👌🏽

**Pull request merged**